agent/consul/config.go

```diff
@@ -160,6 +160,8 @@ type Config struct {
 	// must match a provided certificate authority. This can be used to force client auth.
 	VerifyIncoming bool
 
+	VerifyIncomingRPC bool
+
```

Comment:
Is a new feature/config?

Reply:
See explanation above. I reverted this change.